payroll/models/hr_salary_rule.py

```diff
@@ -179,7 +179,7 @@ def _recursive_search_of_rules(self):
         return [(rule.id, rule.sequence) for rule in self] + children_rules
 
     # TODO should add some checks on the type of result (should be float)
-    def _compute_rule(self, localdict):
+    def _compute_values(self, localdict):
```

Comment:
Is there any benefit in changing the function value? If not i think we should maintain the original one because private modules could be inheriting the function.